### PR TITLE
Replace {{event}} with more specific {{domxref}}

### PR DESCRIPTION
**Pull request by mfuji09**
#### Summary
Replace {{event}} with more specific {{domxref}}

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/htmlinputelement/webkitentries/index.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ document.getElementById("files").addEventListener("change", function(event) {
 });
 ```
 
-Each time a {{event("change")}} event occurs, this code iterates over the selected
+Each time a {{domxref("HTMLElement/chagne_event", "change")}} event occurs, this code iterates over the selected
```
Comment (teoli2003) [inline files/en-us/web/api/htmlinputelement/webkitentries/index.md:67]: Typo!
```suggestion
Each time a {{domxref("HTMLElement/change_event", "change")}} event occurs, this code iterates over the selected
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]
A typo and we are good to go!

File: files/en-us/web/api/htmlinputelement/webkitentries/index.md (hunk 1 above)
Comment (mfuji09) [reply files/en-us/web/api/htmlinputelement/webkitentries/index.md:67]: Oh, Thanks!

**Review by teoli2003 — APPROVED** [review]
Thanks!

**Pull request merged**